### DISCUSSION TITLE
Lowest bitate (10%) is still too high for very slow connections

### DISCUSSION THREAD

**Discussion by tomasz1986** [rustdesk/rustdesk#5231] [General]
### Bug Description

When trying to use RustDesk on a very slow Internet connection (e.g. 1 Mb/s), the screen update and response time are extremely laggy. Setting the bitrate to 10% with custom settings (which seems to translate to 240kb according to the quality monitor) is not enough to overcome the bandwidth limitations. In order for RustDesk to work on such slow connections, even lower bitrate would be required, however it's currently impossible to go lower than 10%. This is despite the fact that when using the bitrate of 10%, the screen quality is still very good, so I'd assume that it should still be possible to lower the bitrate even further simply by decreasing the image quality.

### How to Reproduce

Remotely connect to a device located on a very slow network (e.g. 1 Mb/s upload speed).

### Expected Behavior

Remote control should remain responsive on the expense of deteriorated image quality.

### Operating system(s) on local side and remote side

Windows 10 > Windows 10

### RustDesk Version(s) on local side and remote side

1.2.1

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/5626656/09392c68-3671-486f-84a7-2e521486da19)

**Comment by rustdesk** [comment]
nightly build.

**Reply by mesvam** [reply]
I'm using the nightly and it still seems to be a minimum at 10%?

**Comment by tomasz1986** [comment]
Thank you very much for the update! I didn't want to install the nightly build as then I would need to remember to keep updating it regularly, but I've just tried out the new v1.2.2 release, and the overall responsiveness seems to be much better.

**Comment by lyaaz** [comment]
I still get a 2592 kbps bitrate with a custom setting at 10% (on both stable 1.3.2 and nightly build), which is a bit too high for my connection. Could we reduce the minimum to 5%? @rustdesk

**Reply by lyaaz** [reply]
![image](https://github.com/user-attachments/assets/be7f8de8-4fa2-4b3e-bc59-a3c22b7ac460)

**Reply by 21pages** [reply]
If the controlled side is mac, linux or android, custom quality works on the next connection.We are working on improve this.
